### PR TITLE
Create makefile command 'devstack.destroy' 

### PR DESCRIPTION
**Pull request by jbarciauskas**
Create makefile command 'devstack.clean' to clean images and volumes created by devstack.provision

### CODE REVIEW

**Review by clintonb — COMMENTED** [review]
Is there a way to make this require confirmation? I have intentionally avoided this target to prevent accidental deletion of data.

**Comment by jbarciauskas** [comment]
One option is to change it to "destroy" to indicate the destruction of data and mirror the terminology of vagrant. Another is to create a new bash script to prompt the user. Any preferences? @robrap @jmbowman @clintonb @feanil

**Comment by clintonb** [comment]
I like both. The rename mirrors Vagrant (as you mentioned in HipChat). The script helps prevent accidents.

**Review by clintonb — APPROVED** [review]
Squash your commits.

**Comment by clintonb** [comment]
@jbarciauskas will you update the README, so we can close #41 with this effort?

**Comment by jbarciauskas** [comment]
Done

**Review by clintonb — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,9 @@ help: ## Display this help message
 clone: ## Clone service repos
 	./clone.sh
 
+devstack.destroy:
```
Comment (clintonb) [inline Makefile]: Add help text.

**Pull request merged**